moped-database/migrations/1653067649000_alter_table_moped_proj_milestones/down.sql

```diff
@@ -0,0 +1,19 @@
+-- replace dropped columns
+ALTER TABLE moped_proj_milestones ADD COLUMN milestone_name TEXT;
+
+-- get milestone_name based on milestone_id
```

Comment:
Nice! I like the comments in the migrations a lot for more context. I'm going to revisit the migrations in to my WIP branches. 📝 The aliases also make this more readable. I need to keep that in mind for myself.